src/schema/v2/homeView/sections/QuickLinks.ts

```diff
@@ -146,3 +151,26 @@ export const QUICK_LINKS: Array<NavigationPill> = [
     icon: "FairIcon",
   },
 ]
+
+async function maybeInsertYourBidsLink(
+  links: NavigationPill[],
+  context: ResolverContext
+) {
+  if (isFeatureFlagEnabled("onyx_enable-quick-links-v2")) {
+    const { MyBids } = require("schema/v2/me/myBids")
+    const bids = await MyBids.resolve?.({}, {}, context, {} as any)
+    const hasBids = bids?.active?.length > 0
+    const auctionsIndex = links.findIndex((link) => link.href === "/auctions")
+
+    if (hasBids && auctionsIndex >= 0) {
+      const yourBids: NavigationPill = {
+        title: "Your Bids",
+        href: "/inbox",
+        ownerType: OwnerType.inbox,
+        icon: undefined,
+      }
+      links.splice(auctionsIndex + 1, 0, yourBids)
+    }
```

Comment:
A comment would be helpful to explain where and under what conditions the navigation pill is inserted.